src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -72,6 +73,40 @@ public void Abort()
         public Task CloseOutputAsync(WebSocketCloseStatus closeStatus, string? statusDescription, CancellationToken cancellationToken) =>
             _webSocket!.CloseOutputAsync(closeStatus, statusDescription, cancellationToken);
 
+#if TARGETS_BROWSER
+        public async Task ConnectAsyncCore(Uri uri, CancellationToken cancellationToken, ClientWebSocketOptions options)
```

Comment:
As this is public API it does not make much sense to if-def the method declaration